onnxruntime/core/framework/tunable.h

```diff
@@ -64,18 +75,28 @@ class Op {
   template <typename T>
   explicit Op(T&& c) : callable_{std::make_unique<CallableImpl<T>>(std::forward<T>(c))} {}
   Status operator()(const ParamsT* param) { return (*callable_)(param); }
+  Status IsSupported(const ParamsT* param) { return (*callable_).IsSupported(param); }
```

Comment:
`bool` can only indicate whether it is supported or not. `Status`, however, can also carry the message of why it is not supported. When you want to debug why a kernel is not supported, this will be helpful. Also, it allow us to log the reason in centralized manner.